cfgov/unprocessed/apps/youth-employment-success/css/main.less

```diff
@@ -0,0 +1,43 @@
+@import (reference) '../../../css/main.less';
+
+.a-flush {
+  &__top {
+    margin-top: 0px;
+  }
+
+  &__bottom {
+    margin-bottom: 0px;
+  }
+}
+
+.a-label__flush {
+  &:extend(.a-label);
+  margin-top: 0px;
+}
+
+.block__sub {
+  &-micro {
+    margin-bottom: unit( @grid_gutter-width / @size-i, em );
+    margin-top: unit( @grid_gutter-width / @size-i, em );
+  }
+}
+
+.content_line-bold {
```

Comment:
In this instance, I actually want to preserve the old `comment-line` functionality. The designs call for both a gray dividing line and a black one. Might be better to use a border in the cases of the black line?

Reply:
Ah okay. I think mirroring the background color is fine and makes it consistent with `content_line`. One tweak would be that it appears to be a modifier on `content_line`. We use a form of BEM, `some-block_an-element__a-modifier` (see [here](https://github.com/cfpb/development/blob/a3feffa3485f5e2f721f0c6d77d4965ab7d49c94/standards/css.md#naming-conventions). So `content` is the block and `line` is the element. Since you're essentially modifying the color of the same content line, it should be `.content_line__bold`